packages/vscode-extension/src/debug/taskTerminal/localTunnelTaskTerminal.ts

```diff
@@ -45,6 +54,10 @@ const ngrokEndpointRegex =
   /obj=tunnels name=(?<tunnelName>.*) addr=(?<src>.*) url=(?<endpoint>https:\/\/([\S])*)/;
 const ngrokWebServiceRegex = /msg="starting web service" obj=web addr=(?<webServiceUrl>([\S])*)/;
 const defaultNgrokWebServiceUrl = "http://127.0.0.1:4040/api/tunnels";
+const outputName = {
+  endpoint: "TUNNEL_ENDPOINT",
+  domain: "TUNNEL_DOMAIN",
```

Comment:
Then when to transfer to `BOT_DOMAIN` to be referenced in manifest?

Reply:
Can we use ` env/generate`?
```
  - uses: env/generate # Generate env to .env file
    with:
      envs:
        BOT_DOMAIN: ${{TUNNEL_DOMAIN}}
        BOT_ENDPOINT: ${{TUNNEL_ENDPOINT}}
```

Reply:
Anyway, work for happy path and collect feedback later.

Reply:
Any concern to not directly use `BOT_DOMAIN` and `BOT_ENDPOINT`?